content/print/overview.md

```diff
@@ -6,10 +6,10 @@ draft: false
 author: "people"
 ---
 
-This view allows you to print your images. Because printing is not easy, there are many technical aspects to be taken into account.
+This view allows you to print your images. 
 
-After selecting an image in the [lighttable view](../lighttable/_index.md) you can enter the [print settings](../module-reference/utility-modules/print/print-settings.md) module to adjust printer settings and initiate printing.
+Darktable only scans for printers at startup, so your target printer must be active when darktable starts in order to provide the print module with the target printer's ICC profile. 
```

Comment:
Did you test it yourself?

Reply:
I did!  Several times!!  

I have a colour printer that is normally switched off as I rarely print anything.  When I first looked at the Print view I couldnt load any image into centre panel, so nothing could work.  After some head scratching and checking of Issues in gihub, I managed to understand why.

If I turn my printer on, and let it settle down I can then start dt and see my printer in the dropdown list.  Of course I still have to know to select an image in lighttable after which I get the image in Print view, plus drag and drop from the filmstrip.  And I can print out a colour image just fine.

Reply:
Examining the source code shows that darktable does not scan printers at startup. Therefore, your statement is incorrect.